### PR TITLE
Bhavya Mehta

### PR DESCRIPTION
**Pull request by bhavyamehta4439**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6492"></a>

### CODE REVIEW

**Review by SaraJaoude — DISMISSED** [review]
Please move your file to the `data` folder. 

I would suggest you look at our QuickStart Guide (especially section 7) in Docs: https://linkfree.io/docs/quickstart to help you update this Pull Request.

File: bhavyamehta4439.json{   "name": "Sara Jaoude",   "type": "personal",   "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado" }
Comment (HassanTanveer) [inline bhavyamehta4439.json{   "name": "Sara Jaoude",   "type": "personal",   "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado" }:1]: This file is not necessary.

File: bhavyamehta4439.json
Comment (HassanTanveer) [inline bhavyamehta4439.json:1]: Please move this file the the **data** folder.

The [Quick Start](https://linkfree.io/docs/quickstart) guides you exactly how to do that

**Review by HassanTanveer — CHANGES_REQUESTED** [review]
Please rename PR to:

`data: bhavyamehta4439`

File: bhavyamehta4439.json
Comment (SaraJaoude) [reply bhavyamehta4439.json:1]: Thank you for reviewing @HassanTanveer, but the duplicate feedback is not required and can cause confusion to the author - so I shall mark this comment as `resolved`

File: bhavyamehta4439.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+          with multiple talents."
+  "links": [
+    {
+      "name": "Follow me on Linekdin",
+      "url": "https://www.linkedin.com/in/bhavya-mehta-b6a824226",
+      "icon": "FaLinekdin"
+     }
+    ]
+    "links": [
+    {
```
Comment (SaraJaoude) [inline bhavyamehta4439.json:15]: corrections are needed for the json to pass:
```suggestion
  "bio": "Pursuing Btech in CSE I am creative and cultural with multiple talents",
	"links": [{
			"name": "Follow me on Linekdin",
			"url": "https://www.linkedin.com/in/bhavya-mehta-b6a824226",
			"icon": "FaLinkedin"
		},
		{
```

**Pull request merged**